.goreleaser.yml

```diff
@@ -112,6 +112,16 @@ builds:
       post:
         - cmd: scripts/gon_filepath_editor.sh {{ .Path }} arm64
         - cmd: gon dist/gon_confluent_arm64.hcl
+  - builder: prebuilt
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    goamd64:
+      - v1
+    prebuilt:
+      path: dist/confluent_{{ .Os }}_{{ .Arch }}_{{ with .Amd64 }}_v1{{ end }}/confluent
```

Comment:
Something to keep in mind if we upgrade the Goreleaser version in the future: directory names inside `dist/` aren't guaranteed to be the same between goreleaser versions according to the docs.

It might be safer to parse the path from the `dist/artifacts.json` files of the linux builds; the docs do say that we should always be able to get the correct path from there.

Reply:
> It might be safer to parse the path from the dist/artifacts.json files of the linux builds

Since the path in the dist/artifacts.json file is based on the path in the Docker image, I'll make a note about this and come back to this in the future (since we may or may not still be using Docker)!